src/Workleap.OpenApi.MSBuild/SwaggerManager.cs

```diff
@@ -49,7 +49,7 @@ public async Task InstallSwaggerCliAsync(CancellationToken cancellationToken)
         var retryCount = 0;
         while (retryCount < 2)
         {
-            var result = await this._processWrapper.RunProcessAsync("dotnet", new[] { "tool", "update", "Swashbuckle.AspNetCore.Cli", "--ignore-failed-sources", "--tool-path", this._swaggerDirectory, "--version", SwaggerVersion }, cancellationToken);                
+            var result = await this._processWrapper.RunProcessAsync("dotnet", new[] { "tool", "update", "Swashbuckle.AspNetCore.Cli", "--ignore-failed-sources", "--tool-path", this._swaggerDirectory, "--configfile", Path.Combine(this._openApiToolsDirectoryPath, "nuget.config"), "--version", SwaggerVersion }, cancellationToken);
```

Comment:
I don't see that we specify the full path name for the nuget.config, so what is the current directory when dotnet tool update is executed? Shall we specify the full path name for nuget.config to be sure there's no error such as "nuget.config" does not exists (in the current directory?)

Reply:
Not sure to understand we are referencing the full path 🤔
![image](https://github.com/gsoft-inc/wl-openapi-msbuild/assets/8277266/c72b4932-9180-4424-bf7b-a3e782ffa9b7)

Reply:
And also it this library that create this nuget file at this location: 
![image](https://github.com/gsoft-inc/wl-openapi-msbuild/assets/8277266/81432c74-5be3-4c45-b767-708b25fe976a)